### DISCUSSION TITLE
@quasar/vite-plugin Vite 3 Compatibility

### DISCUSSION THREAD

**Discussion by MrHPotter** [quasarframework/quasar#13958] [Ideas / Proposals] [answered]
Currently the @quasar/vite-plugin  plugin has a peer dependency of vite ^2.0.0 
Thus it can not be used alongside vite 3 out of the box.
I do not know if any additional Changes are required for compatibility other then adding ^3.0.0 to the peer deps as the breaking changes in version 3 are quite far and few between.

Thus i suggest updating the peer dependencies and anything necessary for that change.

**Comment by yusufkandemir** [comment, marked as answer]
This got addressed by #13999 and got released in [@quasar/vite-plugin-v1.1.0](https://github.com/quasarframework/quasar/releases/tag/%40quasar%2Fvite-plugin-v1.1.0) 🚀
